horton/scripts/wpart.py

```diff
@@ -77,25 +76,26 @@ def wpart_slow_analysis(wpart, mol):
         # Prepare solid harmonics on grids.
         grid = wpart.get_grid(index)
         if wpart.lmax > 0:
-            work = np.zeros((grid.size, npure-1), float)
-            work[:,0] = grid.points[:,2] - wpart.coordinates[index,2]
-            work[:,1] = grid.points[:,0] - wpart.coordinates[index,0]
-            work[:,2] = grid.points[:,1] - wpart.coordinates[index,1]
+            work = np.zeros((grid.size, npure - 1), float)
```

Comment:
Just minor note: adding whitespace around arithmetic operators is not a pep8 requirement. It is apparently not a generally accepted style (for some languages). See http://stackoverflow.com/questions/18171560/python-coding-style-is-whitespace-around-operator-needed-in-functions-actual-p